### PR TITLE
Add: Support tssln

### PR DESCRIPTION
**Pull request by sevenc-nanashi**
This PR adds importing/exporting ability for tssln (VoiSona) file.

Closes: #149 

- [x] Import: Basic
- [ ] <s>Import: Pitch</s>
- [x] Export: Basic
- [ ] <s>Export: Pitch</s>

The pitch was too difficult for me, so I don't include in this pR.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 kotlin.code.style=official
 kotlin.js.compiler=ir
 kotlin.daemon.jvmargs=-Xmx4G
+kotlin.js.yarn=false
```
Comment (sevenc-nanashi) [inline gradle.properties:4]: I had to quit yarn to use jsr packages (Kotlin/JS does not recognize .npmrc with yarn)

Comment (sdercolin) [reply gradle.properties:4]: Okay let's just use npm. I'm always uncomfortable with the yarn.lock file as well 😂 
Can we remove those yarn-specific files?

Comment (sevenc-nanashi) [reply gradle.properties:4]: I deleted `yarn.lock` but it spawn again somehow...

Comment (sevenc-nanashi) [reply gradle.properties:4]: Somehow it started to work, so I'm reverting this change.

**Pull request merged**